src/app-data/kinds/ir/irAutoPeakPickingEnhancer.ts

```diff
@@ -20,11 +20,10 @@ export interface AutoPeakPickingOptions extends GSDOptions {
   minPeakWidth?: number;
 }
 
-/** Based on a x value we will return a peak*/
 export function irAutoPeakPickingEnhancer(
   measurement: IrMeasurement,
   options: AutoPeakPickingOptions = {},
-) {
+): void {
```

Comment:
Is this necessary to specify the "void" return type ?

Reply:
This is to avoid returning something by mistake. In general it's better to specify the return type of functions.